Modules/getpath.c

```diff
@@ -228,10 +228,17 @@ getpath_isxfile(PyObject *Py_UNUSED(self), PyObject *args)
     if (path) {
 #ifdef MS_WINDOWS
         const wchar_t *ext;
+#ifdef MS_WINDOWS_GAMES
+        ext = (cchPath >= 4) ? path + cchPath - 4 : NULL;
+#endif
         DWORD attr = GetFileAttributesW(path);
         r = (attr != INVALID_FILE_ATTRIBUTES) &&
             !(attr & FILE_ATTRIBUTE_DIRECTORY) &&
+#ifdef MS_WINDOWS_GAMES
+            (ext != NULL) &&
+#else
             SUCCEEDED(PathCchFindExtension(path, cchPath + 1, &ext)) &&
+#endif
```

Comment:
@zooba, `PathCchFindExtension()` isn't needed here. The simpler approach that's implemented for `MS_WINDOWS_GAMES` is sufficient.